tabular/src/autogluon/tabular/models/lgb/hyperparameters/parameters.py

```diff
@@ -149,3 +149,35 @@ def get_param_softclass_baseline_custom(num_classes):
     params = get_param_multiclass_baseline_custom(num_classes)
     params.pop('metric', None)
     return params.copy()
+
```

Comment:
consider saving these changes for future PR to avoid confusing users

Reply:
I'd removed them:)